src/components/Stake/CardAccordion.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonIcon,
+  Card,
+  GU,
+  IconUp,
+  IconDown,
+  RADIUS,
+  useTheme,
+} from '@aragon/ui'
+import { Spring, Transition, animated } from 'react-spring/renderprops'
+import styled from 'styled-components'
+
+const CardAccordion = React.memo(function CardAccordion({ card, expansion }) {
```

Comment:
We probably don't need to memoize this component as the parent is taking care of it. Also, how do you feel about a different component name? Accordion to me generally feels like a term for multiple elements in a list. What do you think of `ExpandableCard`?